korge-spine/src/commonTest/kotlin/com/esotericsoftware/spine/BonePositionTest.kt

```diff
@@ -7,51 +7,31 @@ import com.soywiz.korio.async.suspendTest
 import com.soywiz.korio.file.std.resourcesVfs
 import com.soywiz.korio.util.OS
 import kotlin.test.Test
+import kotlin.test.assertEquals
 
 
 class BonePositionTest {
     @Test
     fun test() = suspendTest({ !OS.isJs }) {
+        var testResult = resourcesVfs["BonePositionTest.txt"].readString().lines()
+
         val resource = resourcesVfs["spineboy/spineboy-ess.json"]
-        val json = SkeletonJson(AttachmentLoaderTest())
+        val json = SkeletonJson(object : AttachmentLoader { })
         val skeletonData = json.readSkeletonData(resource)
         val skeleton = Skeleton(skeletonData)
         val bone = skeleton.findBone("gun-tip")
         val fps = 1 / 15f
+        var line = 0
         for (animation in skeletonData.animations) {
             var time = 0f
             while (time < animation.duration) {
                 animation.apply(skeleton, time, time, false, null, 1f, MixBlend.first, MixDirection.`in`)
                 skeleton.updateWorldTransform()
-                println(animation.name + "," + bone!!.worldX + "," + bone.worldY + "," + bone.worldRotationX)
+                assertEquals(testResult[line++],"${animation.name},${bone!!.worldX},${bone.worldY},${bone.worldRotationX}")
+                println("${animation.name},${bone!!.worldX},${bone.worldY},${bone.worldRotationX}")
```

Comment:
The problem with this approach is that if for example if 10 output lines change, the way to update them is way harder  than with a single assert, also even when running fine you are outputing 155 lines.

With a single assert you don't need the print, when using the IDE comparing the strings you can just copy and paste. 
Initially I had problems with resources in tests on multiplatform, specially on JS. Not sure if that's fixed. I usually embed strings because it is easier to find the code and the expectation, but if it works in a separate file, I'm ok with it.

![a](https://user-images.githubusercontent.com/570848/95471502-4ac8bc80-0982-11eb-8cf6-c3342bd88b1a.png)
![b](https://user-images.githubusercontent.com/570848/95471508-4c928000-0982-11eb-8f78-9309577450a6.png)

Reply:
I didn't know about that feature in intelliJ, thanks for the tip. I'll change to a test of one of the animations and assert agains a string instead of a file content